index.html

```diff
@@ -0,0 +1,485 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Playfair+Display&family=Poppins:wght@300;400;500;600;800&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="./styles/style.css" />
+    <script src="main.js" defer></script>
+    <title>Adel Guitoun - Portfolio</title>
+  </head>
+  <body class="bg-decoration">
+    <!-- Header Section -->
+    <header class="header-section flex">
+      <nav class="nav-bar">
+        <button type="button" class="menu-btn">
+          <img src="./assets/icons/menu.svg" alt="menu" />
+          <img class="hidden" src="./assets/menu/cancel.svg" alt="cancel" style="color: red;"/>
+        </button>
+
+        <ul class="nav-list hidden" role="list">
+          <li>
+            <a href="#">Hello</a>
+          </li>
+          <li>
+            <a href="#">Portfolio</a>
+          </li>
+          <li>
+            <a href="#">About</a>
+          </li>
+          <li>
+            <a href="#">Contact</a>
```

Comment:
Kindly ensure that the correct part of the page is displayed when the user clicks any of the mobile menu options, as it is one of the requirements🙏.
<img width="598" alt="image" src="https://user-images.githubusercontent.com/37116322/218260792-85d955b6-8502-4d4d-a8f9-cc44c86521ee.png">

This can be achieved by specifying the id of the section of the webpage in the href attribute. e.g `href='#project-section'`. Adding this to your nav links will greatly improve the user experience as they can simply click the links to get to the part of the page they want👍